dashboard/app/models/user.rb

```diff
@@ -1599,6 +1606,13 @@ def generate_secret_picture
 
   def generate_secret_words
     MAX_SECRET_RESET_ATTEMPTS.times do
+      # Teachers should NOT have a secret words login, as this leads to
+      # a security risk when joining a section created by another teacher
+      if teacher?
+        self.secret_words = nil
```

Comment:
Setting these to `nil` I assume pedantically or necessarily prevents such logins?

Reply:
Right. We really shouldn't allow teachers to have a picture or secret word login. With these set to nil, it's impossible to login with them.

Reply:
Sweet. Thanks. That makes 100% sense.

Reply:
Backing this change out as it potentially makes it easier to brute force a login (by hitting the login endpoint with the secret pic or word value set to nil in the payload)